samples/hosting/multi-hosting/Version_5/Sample/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
```

Comment:
I started from another sample that was based on 5.1, I'll update it to the latest. Thanks.